docs/collaboration/reports/google_summer/2018-gsoc.md

```diff
@@ -0,0 +1,158 @@
+The licence on our repository is CC-BY and these guidelines have already been
+incorporated into other open source neuroimaging projects such as
+[NiBetaSeries](https://nibetaseries.readthedocs.io/en/stable/). We also used the
+[“good-first-issue” principle](https://github.com/bids-standard/bids-starter-kit/blob/main/CONTRIBUTING.md#where-to-start-issue-labels)
+to periodically leave small mistakes in our repository unfixed, such as a typo
+or an undefined term in the glossary. Then, we would send out a request and
+guide a new contributor through the process of using the repository.
+
+As a result, the number of unique visitors grew from less than 5 to peaks of
+over 50 per day.
+
+<br>
+
+:o: **Problem** International project management and communication
+
+:heavy_check_mark: **Solution** The members of this project were located in
+Canada ([Elizabeth DuPre](https://github.com/emdupre)), Britain
+([Kirstie Whitaker](https://github.com/KirstieJane)), the Netherlands
+([Dora Hermes](https://github.com/dorahermes)), the USA
+([Chris Gorgolewski](https://github.com/chrisgorgo) and
+[Chris Holdgraf](https://github.com/choldgraf/)), as well as a collaborator from
+India ([Madhur Tandon](https://github.com/madhur-tandon)). The strategy for this
+problem was to work publicly and to demonstrate best practise in open source
+project leadership. This meant rather than waiting until completion to share
+something, we would continuously post our work in progress and incorporate
+feedback. Specifically, Github milestones and “closing soon” tags on issues were
+helpful to highlight weekly tasks and mark them for review.
+
+Despite these strategies, I still felt that the turning point of the project was
+in June when some of our team finally met in person at the
+[Organization for Human Brain Mapping Hackathon](https://ohbm.github.io/hackathon2018/).
+I was a recipient of a travel award ($500) for this event and it really made a
+huge difference in my effectiveness as maintainer and content creator for BIDS
+Starter Kit. Summarizing an unnecessary meeting into a message is efficient, but
+familiarity is really important too.
+
+## Personal notes
+
+As my first foray into community development and international project handling,
+the toughest part at first was definitely the uncertainty -- a document doesn’t
+have errors and nobody leaves detailed feedback about why they _didn’t_ feel
+welcomed. But I’m glad to have finally learned the natural continuation of what
+happens as a tool reaches maturity. To every mentor, collaborator,
+administrator, and user that has been a part of this project so far, thank you
+for all the help and feel free to message because I’ll still be staying
+involved!
+
+<br>
+
+Patrick J. Park
+
+Senior undergraduate student in Electrical Engineering (BESc) at Western
+University, Canada
+
+Email: `parkpatrickj@gmail.com`
+
+Github: [Park-Patrick](https://github.com/Park-Patrick)
```

Comment:
```suggestion
Github: [Park-Patrick](https://github.com/Park-Patrick)

Note: [INCF](https://incf.org] was the supporting umbrella organization for this BIDS Google Summer of Code project. 

```
edit to credit -- i confirmed with Elizabeth Dupre on Dec. 7 that this credit should be added.